app.js

```diff
@@ -43,7 +46,28 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(compression());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(publicDirectory));
+
+if (app.get('env') !== 'development') {
+  var livereload = require('livereload');
+  var connectLiveReload = require('connect-livereload');
+
+  // livereload configuration
+  var liveReloadServer = livereload.createServer();
+  liveReloadServer.watch(publicDirectory);
+  liveReloadServer.server.once("connection", () => {
+    setTimeout( () =>{
+    liveReloadServer.refresh("/");
+    }, 100);
+  })
+
+  app.use(connectLiveReload({
+    port: 35729
```

Comment:
Any particular reason for this value? Could or should there be an option to override this by using an environment variable?

Reply:
Or maybe this value could be read from the config file as we do not seem to use environment variables much hut that file we have.

Reply:
I took it from the example https://www.npmjs.com/package/connect-livereload#connectexpress-example. Anyway,  I think that is a default port. I also read that changing the port is not officially supported. 

Yes, we could use an environment variable for storing the port.

Reply:
I’m fine with this if the port is not typically changed or even if not well supported.

Reply:
If this hardcoded port is the only officially supported one, then it makes sense to hardcode it. This is only used in local development environment anyway.